data_hub_api/docmaps/v2/sql/docmaps_index.sql

```diff
@@ -233,7 +238,19 @@ t_preprint_published_at_date_and_tdm_path AS(
     result.manuscript_id,
     result.long_manuscript_identifier,
     biorxiv_medrxiv_api_response.date AS preprint_published_at_date,
-    tdm.tdm_path
+    CASE 
+      WHEN result.preprint_doi LIKE '10.1101/%'
+        THEN tdm.tdm_path 
+      WHEN result.preprint_doi LIKE '10.21203/rs%' or result.preprint_doi LIKE '%arXiv%'
+        THEN CONCAT('s3://prod-elife-epp-meca/', 
```

Comment:
I believe you can pass in multiple parameters to `CONCAT`?

e.g.

```sql
CONCAT(
  's3://prod-elife-epp-meca/',
  result.preprint_doi_without_version,
  '-v',
  result.preprint_version,
  '-meca.zip'
)

Reply:
can we, ah that is good!